### DISCUSSION TITLE
Sentry APIs for offline use cases Apps

### DISCUSSION THREAD

**Discussion by Sampath-Lokuge** [getsentry/sentry-capacitor#190] [Q&A] [answered]
Hi,

We would like to use the Sentry Capacitor plugin with the Ionic/Angular native app. But this is an offline-first app. So I think the Sentry plugin doesn't support such use cases.

So my question here is this. Do you have any APIs to send errors and see those through Sentry dashboard later? i.e. I can send those offline errors to the Sentry APIs when the user has internet. I'll log those errors on SQLite db until that stage. And if the user has internet then I can use normal online workflow with the Sentry capacitor plugin. Please let me know your feedback here. Thanks!

**Comment by HazAT** [comment, marked as answer]
If you use the native SDKs underneath (iOS/Android) it comes with offline caching out of the box, you don't have to do anything.

**Reply by Sampath-Lokuge** [reply]
Sorry, I didn't get this? If I'll use https://github.com/getsentry/sentry-capacitor plugin then will it have an offline feature by default? I have the Ionic/Angular Native mobile app.

**Reply by lucas-zimerman** [reply]
Sentry Capacitor is bundled with the native SDKs (for Android and iOS) so offline caching will work out of the box.

**Reply by Sampath-Lokuge** [reply]
Oh.. this is great. Thanks a lot!
